### ISSUE TITLE
The output image is strange : train.py or predict.py problem?

### ISSUE THREAD

**Issue by t19b205d** [milesial/Pytorch-UNet#407]
Hello!
![pred_22221_6b9f942956941162fa97](https://user-images.githubusercontent.com/116537921/204721108-84ab9afc-db3f-4abf-b969-ad55db2bddd3.png)

I trained

**Comment by milesial** [comment]
Can you share the pretrained model along with the input image that results in these masks?
